lib/templates/createDevice.json

```diff
@@ -89,8 +93,7 @@
                 },
                 "expression": {
                   "description": "Optional expression for measurement transformation",
-                  "type": "string",
-                  "pattern": "^([^<>;'=]+)+$"
+                  "type": "string"
```

Comment:
I understand that in the case of JEXL the values of the exprssion can not be easlity expressed as a pattern, so better to remove that constraint. Is my interpretation correc t?

Reply:
correct

Reply:
NTC